src/main/webapp/jsp/studentCourseDetails.jsp

```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript" src="/js/student.js"></script>
 </c:set>
 
-<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="TEAMMATES - Student">
+<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="TEAMMATES - Student" jsIncludes="${jsIncludes}">
```

Comment:
Strangely so, because `student.js` is supposed to contain all scripts common to student pages but right now our js code are really disorganized...